docs/src/timearray.md

```diff
@@ -0,0 +1,55 @@
+# The `TimeArray` time series type
+
+The `TimeArray` time series type is defined here
+(with inner constructor code removed for readability):
+
+```julia
+struct TimeArray{T,N,D<:TimeType,A<:AbstractArray} <: AbstractTimeSeries
+  timestamp::Vector{D}
```

Comment:
Should use the Julia standard of 4 spaces for indentation

Reply:
👌